src/hdfmap/nexus.py

```diff
@@ -21,14 +21,16 @@
 logger = create_logger(__name__)
 
 
-def check_nexus_class(hdf_group: h5py.Group, nxclass: bytes) -> bool:
+def check_nexus_class(hdf_group: h5py.Group, nxclass: str) -> bool:
     """
     Check if hdf_group is a certain NX_class
-    :param hdf_group:
-    :param nxclass:
-    :return:
+    :param hdf_group: hdf or nexus group object
+    :param nxclass: str name in NX_class attribute
+    :return: True/False
     """
-    return True if hdf_group and NX_CLASS in hdf_group.attrs and hdf_group.attrs[NX_CLASS] == nxclass else False
+    return hdf_group and NX_CLASS in hdf_group.attrs and hdf_group.attrs.get(NX_CLASS) and (
```

Comment:
Use `group_class := hdf_group.attrs.get(NX_CLASS) is not None` instead of `in` check